cmake/modules/FindKeyFinder.cmake

```
@@ -82,5 +84,12 @@ if(KeyFinder_FOUND)
         INTERFACE_COMPILE_OPTIONS "${PC_KeyFinder_CFLAGS_OTHER}"
         INTERFACE_INCLUDE_DIRECTORIES "${KeyFinder_INCLUDE_DIR}"
     )
+    is_static_library(KeyFinder_IS_STATIC KeyFinder::KeyFinder)
+    if(KeyFinder_IS_STATIC)
+      find_package(FFTW REQUIRED)
+      set_property(TARGET Chromaprint::Chromaprint APPEND PROPERTY INTERFACE_LINK_LIBRARIES
```

```suggestion
      set_property(TARGET KeyFinder::KeyFinder APPEND PROPERTY INTERFACE_LINK_LIBRARIES
```
Is this really needed though? I think we can get rid of this whole file because libkeyfinder ships a CMake config module.

---

ping